src/types/index.ts

```diff
@@ -102,6 +103,7 @@ declare global {
     getServicesStatus: (name: ServiceName) => ServiceStatus;
     loadTables(signal: AbortSignal): Promise<ImportedTable[]>;
     removeTable(tableName: string, signal: AbortSignal): Promise<void>;
+    buildSampleCardEncoded(tableName: string, aidColumns: string[], signal: AbortSignal): Promise<string>;
```

Comment:
The return type is weird. `buildNewSQLPayload` returns an object?

Reply:
ah, good catch, I forgot to clean the names, or maybe I somehow overwrote it. I wanted to call it `buildInitialQueries` both here and in metabase api